libeos-updater-util/extensions.h

```diff
@@ -75,6 +75,7 @@ struct _EosExtensions
 
   GBytes *summary;
   GBytes *summary_sig;
+  guint64 summary_modification_time_unix;
```

Comment:
I would call this `summary_modification_time_secs` to make it clear what the units are. Maybe add a comment after: `/* since the Unix epoch */`.